feg/gateway/services/testcore/hss/servicers/ai.go

```diff
@@ -87,7 +89,10 @@ func (srv *HomeSubscriberServer) setLteAuthNextSeq(subscriber *lteprotos.Subscri
 // authentication information request (AIR) message. It populates AIA with all of the mandatory fields
 // and adds the authentication vectors.
 func (srv *HomeSubscriberServer) NewSuccessfulAIA(
-	msg *diam.Message, sessionID datatype.UTF8String, vectors []*crypto.EutranVector) *diam.Message {
+	msg *diam.Message,
+	sessionID datatype.UTF8String,
+	vectors []*crypto.EutranVector,
+	utranVectors []*crypto.UtranVector) *diam.Message {
```

Comment:
for error handling, is it possible to get both empty vectors and utranVectors? Maybe if bot are empty we should trigger an error or a log message warning about this

Reply:
It shouldn't be possible, we generate eutran/utran vectors and will make as many as requested. If zero eutran and zero utran vectors are requested, ValidateAIR() below should error out before call to  NewSuccessfulAIA()